### PR TITLE
7 post/api/reviews/review id/comments

### PR DESCRIPTION
**Pull request by kontsio1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -96,4 +96,28 @@ describe("5. GET /api/reviews:review_id", () => {
         expect(msg).toBe("Not Found!");
       });
   });
-});
\ No newline at end of file
+});
+
+describe.only("7. POST /api/reviews/:review_id/comments", () => {
+  test("status:200, responds with the posted comment object", () => {
```
Comment (Dutton1990) [inline __tests__/app.test.js:102]: wrong status in description

File: controllers/controller.js [hunk 2]
```diff
@@ -26,3 +27,14 @@ exports.getReview = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postComment = (req, res, next) => {
+  const {review_id} = req.params
+  const author = req.body.username
+  const body = req.body.body
+  console.log(author, review_id, body)
```
Comment (Dutton1990) [inline controllers/controller.js]: dont forget to remove console logs!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -120,4 +120,45 @@ describe.only("7. POST /api/reviews/:review_id/comments", () => {
...
+    }
+    return request(app)
+    .post("/api/reviews/5/comments")
+    .send(newComment)
+    .expect(400)
+    .then((res)=>{
+      const msg = res.body.msg
+      expect(msg).toBe("Very Bad Request!")
+    })
+  })
```
Comment (Dutton1990) [inline __tests__/app.test.js:149]: this would be a 404 because the username could not be found, rather than a 400 bad request

**Review by Dutton1990 — CHANGES_REQUESTED** [review]
Hey Konstantinos, a few suggested changes above 😄 

Another test you can do is for review/banana/comments for example, this should be a 400! 

Once you've made those changes send it over to us again tomorrow please!

File: models/model.js [hunk 4]
```diff
@@ -36,3 +36,9 @@ exports.selectReview = (review_id) => {
       }
     });
 };
+
+exports.insertComment = (review_id, author, body) => {
+  return db.query(`INSERT INTO comments (author, review_id, body) VALUES ($1, $2, $3) RETURNING *`, [author, review_id, body]).then((data)=>{
```
Comment (lewispricey) [inline models/model.js:41]: Nice use of parameterization to prevent SQL injection!

**Review by lewispricey — APPROVED** [review]
Great testing updates! really thorough in testing every possibility of a bad user input.

Good to merge 🚀

**Pull request merged**